_includes/v21.2/app/init-bank-sample.md

```diff
@@ -1,25 +1,26 @@
 
 To initialize the example database, use the [`cockroach sql`](cockroach-sql.html) command to execute the SQL statements in the `dbinit.sql` file:
 
+<div class="filter-content" markdown="1" data-scope="cockroachcloud">
+
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-cat dbinit.sql | cockroach sql --url "<connection-string>"
+cat dbinit.sql | cockroach sql --url '<connection-string>'
```

Comment:
This connection string requires double quotes, I'm pretty sure, as we include environment variables in the connection string now.

Reply:
Good catch. Done.